homeassistant/components/roku/browse_media.py

```diff
@@ -0,0 +1,144 @@
+"""Support for media browsing."""
+
+from homeassistant.components.media_player import BrowseMedia
+from homeassistant.components.media_player.const import (
+    MEDIA_CLASS_APP,
+    MEDIA_CLASS_APPS,
+    MEDIA_CLASS_CHANNEL,
+    MEDIA_CLASS_CHANNELS,
+    MEDIA_CLASS_DIRECTORY,
+    MEDIA_TYPE_APP,
+    MEDIA_TYPE_APPS,
+    MEDIA_TYPE_CHANNEL,
+    MEDIA_TYPE_CHANNELS,
+)
+
+CONTENT_TYPE_MEDIA_CLASS = {
+    MEDIA_TYPE_APP: MEDIA_CLASS_APP,
+    MEDIA_TYPE_APPS: MEDIA_CLASS_APPS,
+    MEDIA_TYPE_CHANNEL: MEDIA_CLASS_CHANNEL,
+    MEDIA_TYPE_CHANNELS: MEDIA_CLASS_CHANNELS,
+}
+
+PLAYABLE_MEDIA_TYPES = [
+    MEDIA_TYPE_APP,
+    MEDIA_TYPE_CHANNEL,
+]
+
+EXPANDABLE_MEDIA_TYPES = [
+    MEDIA_TYPE_APPS,
+    MEDIA_TYPE_CHANNELS,
+]
+
+
+def build_item_response(coordinator, payload):
+    """Create response payload for the provided media query."""
+    search_id = payload.get("search_id")
```

Comment:
Please use `dict[key]` when we know the key is in the dict.
```suggestion
    search_id = payload["search_id"]
```